api/src/ginger_calls.rs

```diff
@@ -1,85 +1,117 @@
 use algebra::{fields::{
-    mnt4753::{Fr, Fq as ScalarFieldElement}, PrimeField
+    tweedle::{Fq as Fr, Fr as ScalarFieldElement}, PrimeField
```

Comment:
Mmm you should sync your concrete types with the one used in zendoo-mc-cryptolib and zendoo-cctp-lib: I fear that in both of them tweedle::Fr is used as FieldElement and not Fq as you did

Reply:
If all the types are not synced, the integration tests between MC and SC will never pass (they will read different fields)

Reply:
I have changed the FieldElement from Fq to Fr